src/api/userApi.ts

```diff
@@ -29,7 +29,7 @@ export const getCurrentAuthenticatedUser = () => {
       cognitoUser?.getSession(async (error: any, session: CognitoUserSession) => {
```

Comment:
prøvde å spesifisere `const cognitoUser : CognitoUser` over som ga autocomplete på noen funksjoner, men da klagde den også på at enkelte ting ikke er der. feks var `cognitoUser.attributes` ugyldig, som egentlig gir mening siden vi kan hente  `attributes` direkte via andre argument i callbacket. Men attributes der er merkelig nok ikke på samme format som det man får fra `cognitoUser.attributes`. Måtte legge inn et tomt callback på linje 32 for at TS skulle bli fornøyd også. Har en liten mistanke om at vi blander ny og gammel flyt for cognito, men whatevz. if it twerks it twerks.

tl;dr: blir litt omskrivinger hvis vi skal ta inn typer her :/

Reply:
Her er definisjonen av CognitoUser for spesielt interesserte: https://github.com/aws-amplify/amplify-js/blob/master/packages/amazon-cognito-identity-js/index.d.ts#L63

Reply:
Mulig det er en forskjell på om man bruker hele pakken eller om man bare henter enkeltmoduler: https://github.com/aws-amplify/amplify-js#installation

Reply:
husker vi brukte litt tid for å finne riktig fremgangsmåte på dette. det manglet en del dokumentasjon da vi satte det opp.